### ISSUE TITLE
I can pay invoices on my mainnet LND node, but others cannot pay my invoices

### ISSUE THREAD

**Issue by gg2001** [lightningnetwork/lnd#2629]
I have an LND node running the 0.5.2-beta release on a 2 core, 8GB ram Ubuntu 16.04 VM. This is my node: https://1ml.com/node/02d5efae50c1cdc6011e40b74f061ea4f444088865ec14a59e4ee6cf0d71c842fc

I used the Raspibolt tutorial to setup this node. It appears to be working properly, however when trying to receive payments it doesn't work. I can make channels and stuff, the only issue is being unable to receive payments.

For example, I have a post on [yalls](https://yalls.org/articles/97d67df1-d721-417d-a6c0-11d793739be9:0965AC5E-56CD-4870-9041-E69616660E6F/23c39080-3748-4cf6-a0c4-e1c5c0811e43). I have paid for posts, it works perfectly. However when I try to withdraw whatever I have earned through yalls, it says payout failed. When I try again it says invoice is for too much, although that is false. I can confirm I have entered the right node public key when creating the post as I tried my invoices on other yalls posts and it clearly says the node pubkey is different. [Here](https://imgur.com/a/59Obt7l) is an imgur album demonstrating this.

I have also tested my invoices with BlueWallet, and it says LND api failure 7. [Here](https://imgur.com/a/oseWH4G) is an imgur album showing this. I think the problem here is my node. Has anyone encountered this problem, or know of a solution to it?

TLDR: I can do everything with my LND Node, except receiving payments on invoices I make

**Comment by gg2001** [comment]
I did not have any incoming connections with enough liquidity. I got some and it is working now.
